packages/flutter_form_builder/test/form_builder_tester.dart

```diff
@@ -20,3 +24,5 @@ void formFieldDidChange(String fieldName, dynamic value) {
 }
 
 dynamic formValue(String name) => _formKey.currentState!.value[name];
+dynamic formInstantValue(String name) =>
```

Comment:
Is it possible to use generics instead of dynamic? 
```suggestion
T formValue(String name) => _formKey.currentState!.value[name];
T formInstantValue(String name) =>
```